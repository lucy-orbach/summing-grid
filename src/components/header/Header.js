import React from 'react';
import styles from 'styles/layout.module.css';
import types from 'styles/typography.module.css';


const Header = () => {
	console.log('header');
	return(
	<header className={styles.flex_centered} data-test="header">
		<h1 className={types.header_title}>Summing<span>Grid</span></h1>
	</header>
)};

export default Header;