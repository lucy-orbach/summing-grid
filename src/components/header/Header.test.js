import React from 'react';
import 'setupTests';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from 'components/header/Header';

describe('<Header />', () => {

	it('should render Header component ', () => {
		const wrapper = shallow(<Header />);
		expect( wrapper.find('[data-test="header"]').exists()).toBe(true);
	});

	it('matches the snapshot', () => {
		const tree = shallow(<Header />);
		expect(toJson(tree)).toMatchSnapshot()
	});

});
