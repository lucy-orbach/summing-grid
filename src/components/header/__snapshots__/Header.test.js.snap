// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches the snapshot 1`] = `
<header
  className="flex_centered"
  data-test="header"
>
  <h1
    className="header_title"
  >
    Summing
    <span>
      Grid
    </span>
  </h1>
</header>
`;
